### PR TITLE
added program of print numbers from 1 to n

### LINKED ISSUE #5207: Print numbers from 1 to n
### Description

Write a program to print numbers from 1 to n

```
Input  : 5
Output : 1 2 3 4 5
```

### Tracking Issues

- [ ] Write a C program to print numbers from 1 to n
- [ ] Write a C# program to print numbers from 1 to n
- [ ] Write a C++ program to print numbers from 1 to n
- [ ] Write a Dart program to print numbers from 1 to n
- [ ] Write a F# program to print numbers from 1 to n
- [ ] Write a GO program to print numbers from 1 to n
- [ ] Write a Haskell program to print numbers from 1 to n
- [ ] Write a Java program to print numbers from 1 to n
- [ ] Write a JavaScript program to print numbers from 1 to n
- [ ] Write a Julia program to print numbers from 1 to n
- [ ] Write a Kotlin program to print numbers from 1 to n
- [ ] Write a Perl program to print numbers from 1 to n
- [ ] Write a PHP program to print numbers from 1 to n
- [ ] Write a Python program to print numbers from 1 to n
- [ ] Write a R program to print numbers from 1 to n
- [ ] Write a Ruby program to print numbers from 1 to n
- [ ] Write a Rust program to print numbers from 1 to n
- [ ] Write a Scala program to print numbers from 1 to n
- [ ] Write a Swift program to print numbers from 1 to n
- [ ] Write a TypeScript program to print numbers from 1 to n

### LINKED ISSUE #5216: Write a JavaScript program to print numbers from 1 to n
### Description

Write a JavaScript program to print numbers from 1 to n

```
Input  : 5
Output : 1 2 3 4 5
```
<details>
<summary>How to contribute</summary>

- Save the solution in `PrintNumbersFrom1ToN.js` file
- Add `PrintNumbersFrom1ToN.js` file in `program/print-numbers-from-1-to-n` folder

</details>

### PR DESCRIPTION
**Pull request by shauryag2002**
### Why:

Closes codinasion/codinasion#3871

### CODE REVIEW

**Review by anandfresh — CHANGES_REQUESTED** [review]
You have two programs combined in this PR . 
Can you add the issue ID for the other program 'FindSumOfSquaresOfNumbers.js'.

File: program/program-to-find-sum-of-all-numbers-in-array/FindSumOfSquaresOfNumbers.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+let n=prompt("input the number")
+let b=[];
+let count=0;
+for(let i=0;i<n;i++){
+    b.push(Number(prompt("Number please")))
+
```
Comment (anandfresh) [inline program/program-to-find-sum-of-all-numbers-in-array/FindSumOfSquaresOfNumbers.js:6]: why prompting inside the loop ?

**Review by anandfresh — CHANGES_REQUESTED** [review]

File: program/program-to-find-sum-of-all-numbers-in-array/FindSumOfSquaresOfNumbers.js (hunk 1 above)
Comment (harshraj8843) [reply program/program-to-find-sum-of-all-numbers-in-array/FindSumOfSquaresOfNumbers.js]: for elements

**Review by harshraj8843 — APPROVED** [review]
Thanks for contributing 👍🏻

**Review by anandfresh — APPROVED** [review]

**Pull request merged**